src/Schema/AbstractSchema.php

```diff
@@ -409,6 +409,7 @@ protected function getColumnPhpType(ColumnSchemaInterface $column): string
             SchemaInterface::TYPE_SMALLINT => SchemaInterface::PHP_TYPE_INTEGER,
             SchemaInterface::TYPE_INTEGER => SchemaInterface::PHP_TYPE_INTEGER,
             SchemaInterface::TYPE_BIGINT => SchemaInterface::PHP_TYPE_INTEGER,
+            SchemaInterface::TYPE_BIT => SchemaInterface::PHP_TYPE_INTEGER,
```

Comment:
In MSSQL BIT is 1, 0, or NULL only. Can it be problem?

Reply:
May be use name "BIT_STRING"?

Reply:
The changes are only for `yiisoft/db-pgsql`

For `yiisoft/db-mssql` it is as before:

https://github.com/yiisoft/db-mssql/blob/master/src/Schema.php#L74C9-L74C38
```php
'bit' => self::TYPE_SMALLINT,
```

I do not know any issues with type `bit` in MSSQL but if it is required it is possible to change it to `'bit' => self::TYPE_BIT,` and cast `bit` value in `ColumnSchema::dbTypecast()` or `ColumnSchema::phpTypecast()`.

Reply:
As it is not a global `type` that behaves the same for all `dbms`, i think it would be a good idea to map it in `db-pgsql`, and each `dbms` will have its own behavior, as it should be.